api/utils/cfApiClient.js

```diff
@@ -247,11 +247,28 @@ class CloudFoundryAPIClient {
       (acc, current) => ({ ...acc, [current.name]: current.value }),
       {}
     );
-    const encryptedParams = Encryptor.encrypt(
-      JSON.stringify(commandParams),
-      encryption.key
+    const encryptedParams = Encryptor.encryptObjectValues(
+      commandParams,
+      encryption.key,
+      {
+        onlyEncryptKeys: [
+          'STATUS_CALLBACK',
+          'GITHUB_TOKEN',
+          'AWS_ACCESS_KEY_ID',
+          'AWS_SECRET_ACCESS_KEY',
+          'BUCKET',
+        ],
+      }
     );
-    const command = `cd app && python main.py -p '${encryptedParams.ciphertext}'`;
+    const commandParamsString = JSON.stringify(encryptedParams);
+
+    if (commandParamsString.length >= 4097) {
```

Comment:
this check should probably be against 4069 because it also counts the ~28 characters from the command template string below against the limit